.env.example

```
@@ -65,3 +65,6 @@ REACT_APP_REWARDS_BANNER_WARNING=
 
 # Define Hardhat Infura ID for testing
 HARDHAT_INFURA_ID= 
+
+# Define a configurable message to display to users in environment.
+REACT_APP_GENERAL_MAINTENANCE_MESSAGE = Message to display in top banner.
```

Is there a limit to this message length we should comment about?
```suggestion
REACT_APP_GENERAL_MAINTENANCE_MESSAGE=Message to display in top banner.
```

---

Also i think you're inserting an accidental " " char at the beginning of the string